cmd/startApi.go

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright © 2023 NAME HERE <EMAIL ADDRESS>
+*/
+package cmd
+
+import (
+	"github.com/cloud-native-toolkit/itzcli/api"
+	"github.com/spf13/cobra"
+)
+
+// startCmd represents the start command
+var startCmd = &cobra.Command{
+	Use:   "start",
```

Comment:
Try to make sure to update the documentation and make it real, as its printed out to the user when they click help.